package.json

```diff
@@ -83,6 +88,7 @@
     "extract-zip": "1.5.0",
     "fetch-mock": "4.6.0",
     "glob": "7.0.5",
+    "isomorphic-fetch": "2.2.1",
```

Comment:
How were we not using this previously? It's in our mocha tests, but not sure if that was being used. Was this an implicit dep?